### DISCUSSION TITLE
No valid solidity source files found.

### DISCUSSION THREAD

**Discussion by soyaya** [Cyfrin/security-and-auditing-full-course-s23#36] [Q&A]
I am using windows and i can't get my report, any help?

**Comment by PatrickAlphaC** [comment]
Hi! Thanks for making this discussion. There isn't enough information here for us to help you out.

What is the exact command your running, and error you're running into?

**Reply by cromewar** [reply]
Also, if you are running WSL (as you should if you are on windows), the report should appear on the WSL root folder of your project, but @PatrickAlphaC is right we need more context on this can you please read; https://github.com/Cyfrin/security-and-auditing-full-course-s23/blob/main/how-to-ask-a-question.md and reformulate?

**Reply by soyaya** [reply]
Ok, i will ask properly. I appreciate all your input.  Though i have not gotten around it.

**Comment by Rushikesh0125** [comment]
You may try using command palette. [ctrl+shift+p] 
type solidity metrics -> Report metrics for all open workspaces 
<img width="485" alt="image" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/85375791/15a327bc-4d8e-4b5c-b7c9-c9f445b28bac">

**Comment by EngrPips** [comment]
@soyaya  do you get this issue resolved already ?
